autoload/neomake/makers/ft/text.vim

```diff
@@ -27,3 +35,51 @@ function! neomake#makers#ft#text#writegood() abort
                 \ 'postprocess': function('neomake#makers#ft#text#PostprocessWritegood'),
                 \ }
 endfunction
+
+let s:languagetool_fallback_language = 'auto'
+" Public API: https://languagetool.org/api
```

Comment:
404.
Should be http://wiki.languagetool.org/public-http-api probably.

Reply:
That is the actual value you can set the variable. I'll rephrase it.

See https://github.com/somini/neomake/blob/a52514fe3f89348026280efc9a9714820c43db59/autoload/neomake/makers/ft/text/languagetool.py#L80